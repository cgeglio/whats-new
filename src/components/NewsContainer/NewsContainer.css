.newsContainer {
  display: flex;
  flex-wrap: wrap;
  grid-area: container;
}
