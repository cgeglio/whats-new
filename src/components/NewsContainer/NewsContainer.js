import React from 'react';
import './NewsContainer.css'
import NewsArticle from '../NewsArticle/NewsArticle'

const NewsContainer = (props) => {
  return <NewsArticle
  image = {props.newsItem.img}
  headline = {props.newsItem.headline}
  description = {props.newsItem.description}
  id = {props.newsItem.id}
  />
};

export default NewsContainer;
