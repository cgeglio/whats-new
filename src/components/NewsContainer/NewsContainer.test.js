import React from 'react';
import { shallow } from 'enzyme';
import NewsContainer from './NewsContainer';

describe('NewsContainer', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<NewsContainer
      news={[
        {topic: "local", articles: []},
        {topic: "entertainment", articles: []}
      ]}
    />)
  expect(wrapper).toMatchSnapshot();
  })
})
