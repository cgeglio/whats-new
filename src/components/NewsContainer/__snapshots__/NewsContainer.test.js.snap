// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsContainer should match the snapshot with all data passed in correctly 1`] = `
<section
  className="newsContainer"
>
  <NewsArticle
    key="1580346982113-undefined"
  />
  <NewsArticle
    key="1580346982113-undefined"
  />
</section>
`;
